edge/pkg/metamanager/process.go

```diff
@@ -267,12 +257,16 @@ func (m *metaManager) processUpdate(message model.Message) {
 	case cloudmodules.PolicyControllerModuleName:
 		resp := message.NewRespByMessage(&message, OK)
 		sendToCloud(resp)
-	case modules.MetaManagerModuleName:
-		// Process the update message from MetaManager(MetaServer)
-		// which is used to update the device in edge node.
-		sendToTwin(&message)
-		resp := message.NewRespByMessage(&message, OK)
-		sentToMetamanager(resp, message.IsSync())
+	case metaserver.MetaServerSource:
+		switch resType {
+		case dtcommon.ResourceDevice:
+			resp := message.NewRespByMessage(&message, OK)
+			beehiveContext.SendResp(*resp)
+			message.SetRoute(modules.MetaGroup, modules.DeviceTwinModuleName)
+			beehiveContext.Send(modules.DeviceTwinModuleName, message)
```

Comment:
According to the design in #5901, our purpose is to directly operate resources such as `pod` ,`devices` at the edge, but why are only device resources considered here?